core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -502,6 +506,8 @@ void ExHbaseAccessTcb::freeResources()
      NADELETEBASIC(directRowBuffer_, getHeap());
   if (colVal_.val != NULL)
      NADELETEBASIC(colVal_.val, getHeap());
+  if (hdfsClient_ != NULL) 
+     NADELETE(hdfsClient_, HdfsClient, getHeap());
 }
```

Comment:
I am wondering if there is a need to delete the new introduces loggingFileName_ here?

Reply:
Yes. It is a good catch. I will fix this too in my next commit.  This won't cause memory leak as such because the heap is destroyed.